manifests/init.pp

```diff
@@ -183,9 +134,33 @@
     '/usr/share/containers/oci/hooks.d': ;
   }
 
+  if $manage_cilium {
```

Comment:
For me this belongs to cilium, can this move to the cilium class?

Reply:
the cilium class might only be loaded on one or all of the controllers, but the settings are needed on every host which is doing some networking stuff. which are, if i'm not wrong, all servers? and i don't think we want to load this class on every node and also not have the binary around on every host?